### PR TITLE
First pass at removing old and adding new checks

### PR DESCRIPTION
**Pull request by jeanconn**
First pass at removing old and adding new checks

### CODE REVIEW

File: jobwatch/skawatch.py [hunk 1]
```diff
@@ -74,8 +74,10 @@ def __init__(self, task, maxage=1, dbfile=None, table=None, timekey='tstart'):
 perl_errs = set(('uninitialized value',
                  '(?<!Program caused arithmetic )error',
                  'warn', 'fatal', 'fail', 'undefined value'))
-arc_exclude_errors = [r'warning:\s+\d+\s',
-                      'file contains 0 lines that start with AVERAGE']
+arc_exclude_errors = [
+    r'warning:\s+\d+\s',
+    'file contains 0 lines that start with AVERAGE',
+    'p3 fluence not plotted, error : cannot do a non-empty take from an empty axes.']
```
Comment (taldcroft) [inline jobwatch/skawatch.py:80]: I'm know I'm missing the context, but at first glance this sure *looks* like an error that should not be ignored.

File: jobwatch/skawatch.py [hunk 2]
```diff
@@ -108,18 +110,20 @@ def main():
 
     jws = []
     jws.extend([
-        SkaJobWatch('aca_lts_eval', 2, errors=py_errs),
         SkaJobWatch('aca_hi_bgd_mon', 2, errors=py_errs,
                     filename='/proj/sot/ska/data/aca_hi_bgd_mon/logs/daily.0/aca_hi_bgd.log'),
         SkaJobWatch('acdc', 2, errors=py_errs),
-        SkaJobWatch('aimpoint_mon', 2, errors=py_errs),
+        SkaJobWatch('aimpoint_mon', 2, errors=py_errs,
+                    exclude_errors=['ErfaWarning']),
```
Comment (taldcroft) [inline jobwatch/skawatch.py:117]: Please do remember to revert this change in a few weeks when astropy 4.2 is included. Although it is benign from the processing perspective, it's good to see these in our processing as a precursor to users stumbling into the same problem.

File: jobwatch/skawatch.py (hunk 1 above)
Comment (jeanconn) [reply jobwatch/skawatch.py:80]: Sure.  I just thought we were doing a fine job of ignoring it now so could perhaps continue.

File: jobwatch/skawatch.py (hunk 2 above)
Comment (jeanconn) [reply jobwatch/skawatch.py:117]: In this case, I was thinking that using 'aimpoint_mon' for sensitivity to ErfaWarning is not the right thing.  But if we don't have module testing in cxotime or some such that would clue us in, I can be more sensitive to it.  In this case, because aimpoint_mon is Python 2 code and we're not going to update astropy to 4.2 in Ska2 and I don't know when the aimpoint-mon code will be updated to Ska3,  it seems like the right fix is to shut up the warnings in Python 2 (by either getting a new file, or a live fix to ignore the warnings in aimpoint_mon or astropy in Ska2).

Comment (taldcroft) [reply jobwatch/skawatch.py:117]: With https://github.com/sot/aimpoint_mon/pull/21 merged it will not be *too* far in the future when we do the Py3 transition. Anyway, I don't mind shutting this up now, but I would rather not leave these ERFA suppression hooks in place indefinitely.

File: jobwatch/skawatch.py (hunk 1 above)
Comment (taldcroft) [reply jobwatch/skawatch.py:80]: Not paying attention at the moment is slightly different from permanently ignoring.

File: jobwatch/skawatch.py (hunk 2 above)
Comment (jeanconn) [reply jobwatch/skawatch.py:117]: Could I just plop this in the current Ska2 plot_aimpoint.py  or some such?
```
import warnings
from astropy._erfa.core import ErfaWarning
warnings.filterwarnings("ignore", category=ErfaWarning)
```

Comment (jeanconn) [reply jobwatch/skawatch.py:117]: Shutting up the not-flight-managed code directly seems like it would shut things up and have the side-effect that if the problem comes up again (with a ska3 install) we'd see it.

File: jobwatch/skawatch.py (hunk 1 above)
Comment (jeanconn) [reply jobwatch/skawatch.py:80]: Sure.  But hard to figure out which things need to be red at the skawatch level, and which things are OK for verbose output in the email watch logs level, and which things we're just always going to ignore.  Maybe for skawatch we don't care at *all* about checking the arc logs and just want to know all the files were updated sometime recently.

In general, I would like to see more review of warnings and filters in jobwatch as a natural part of making changes to trending applications.

File: jobwatch/skawatch.py (hunk 2 above)
Comment (jeanconn) [reply jobwatch/skawatch.py]: I've cut the ErfaWarning here for now.

**Review by taldcroft — APPROVED** [review]
